lua/entities/gmod_wire_expression2/core/wirelink.lua

```diff
@@ -107,6 +107,17 @@ local function WriteArray(entity, address, data)
 	return free_address
 end
 
+local function writeArraySimple(entity, address, data)
+	local written = 0
+	for index, value in pairs(data) do
+		if type(value) == "number" then
```

Comment:
@Divran can you confirm this is indeed the case?
also, is there type info associated with this part of the table and if not, note that down for the output converter thing we were talking about :)